specs/darwin/crashes.table

```diff
@@ -1,5 +1,5 @@
 table_name("crashes")
-description("Application and System Crash Logs")
+description("Application and System Crash Logs.")
```

Comment:
`Application, System, and Mobile App crash logs.`? At the very least would you mind changing `crash` and `logs` <3